app/middlewares/errors.js

```diff
@@ -6,7 +6,8 @@ const DEFAULT_STATUS_CODE = 500;
 const statusCodes = {
   [errors.DATABASE_ERROR]: 503,
   [errors.DEFAULT_ERROR]: 500,
-  [errors.EXTERNAL_SERVICE_ERROR]: 502
+  [errors.EXTERNAL_SERVICE_ERROR]: 502,
+  [errors.FIELD_VALIDATION_ERROR]: 422
```

Comment:
It's okey to return a a 4.x.x error. But maybe 422 is not right here.
The definition of 422 is: 
> The request was well-formed but was unable to be followed due to semantic errors
I think the user forgetting the first_name field is not a semantic issue. I would choose 400 for this case. But it's open to discussion

Reply:
> The HyperText Transfer Protocol (HTTP) 422 Unprocessable Entity response status code indicates that the server understands the content type of the request entity, and the syntax of the request entity is correct, but it was unable to process the contained instructions.

I think it is applicable here because content type is correct and not malformed. Is a wellformed JSON, but it makes non sense that an user doesn't contain an email.

Reply:
It's a good point. And it's a really interesting discussion.
Here is a thread discussing it: https://stackoverflow.com/questions/16133923/400-vs-422-response-to-post-of-data
Some think 422 should be use only in cases of xml, based on one protocol. Some think that we could replace "xml" with "json" in the protocol and it works the same. Some name other protocols.
422 it's ok, then :+1: